database.cc

```diff
@@ -1335,7 +1335,7 @@ future<mutation> database::do_apply_counter_update(column_family& cf, const froz
                           regular_columns.end());
 
     auto slice = query::partition_slice(std::move(cr_ranges), std::move(static_columns),
-        std::move(regular_columns), { }, { }, cql_serialization_format::internal(), query::max_rows);
+        std::move(regular_columns), { }, { }, cql_serialization_format::internal());
```

Comment:
We can remove max_rows because of the default value of the parameter?

Reply:
It was the reason but on second thought there is no reason to change it here